packages/client/mutations/StartRetrospectiveMutation.ts

```diff
@@ -7,18 +7,6 @@ graphql`
   fragment StartRetrospectiveMutation_team on StartRetrospectiveSuccess {
     meeting {
       id
-      defaultFacilitatorUserId
-      name
-      meetingMembers {
-        user {
-          id
-          preferredName
-        }
-      }
-    }
-    team {
-      ...TopBarMeetingsActiveMeetings @relay(mask: false)
```

Comment:
if we remove this stuff, how do other team members know a meeting has started?

Reply:
As we're just using the `meetingId` in this file, I didn't think we needed the other fields. The fragment is being used in the meeting subscription so I thought that when the `meetingId` updates, i.e. when the meeting is created, the team subscription updates and lets the other members know it's started

Reply:
not sure i follow 100%, but i think the gotcha here is that the user isn't subscribed to a meeting until they enter the meeting, so if someone is on the dashboard, they only have the team subscription going